streaming/java/dependencies.bzl

```
@@ -14,6 +14,10 @@ def gen_streaming_java_deps():
             "org.slf4j:slf4j-log4j12:1.7.25",
             "org.apache.logging.log4j:log4j-core:2.8.2",
             "org.testng:testng:6.9.10",
+            "log4j:log4j:1.2.17",
+            "org.mockito:mockito_all:1.10.19",
```

```suggestion
            "org.mockito:mockito-all:1.10.19",
```

typo which cause dependency resolution failed.

---

Thanks for the suggestion comment. I'll fix this